content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,135 @@
+
+## 4. Avalanche
+
+   The next level solution for scalability in terms of consensus algorithms is by Avalanche network.
+   In Avalanche, the underlying [sybil control mechanism](https://en.wikipedia.org/wiki/Sybil_attack) is still PoS but there is a different mechanism in place to finalise validating transactions.
+   This is achieved by a particular node randomly subsampling from other nodes multiple times across multiple cycles and eventually arriving at a consensus.
+   This is then recorded in a Directed Acyclic Graph as opposed to a chain of Blocks.
+   This consensus can easily scale at a much higher level compared to Bitcoin/PoW or Ethereum2.0/PoS because of faster agreement of transactions.
+   The issue with Avalanche is similar to what other PoS based networks also have i.e. the economic limit of having a lot of value locked as stake in order to have the PoS running which may not be practical when the network overall value is in trillions of dollars.
+   The minimum staking that is required to become a validator may not be easy for an average person to be a validator without delegating to another validator thereby contributing to centralisation of the network.
+   Thus Avalanche has validators only in the number of thousands and not tens or hundreds of thousands so far.
+   Usually many PoS based networks impose a penalty for malicious validators or validators that aren’t online continuously by slashing which is taking away some of the staked tokens as a punishment.
+   Avalanche doesn’t have this mechanism to encourage more participation as a step towards more decentralisation thereby sacrificing a bit more of security.
+
+   (Doubt: How the rich get richer compounding effect is absent in Avalanche?)
+   (Doubt: How exactly the transactions get written after consensus and how reward distribution happens in Avalanche?)
+
+## 5. Flare
+
+   Flare network combines best of both worlds by making Avalanche more resistant to Sybil attacks by introducing FBA into it.
+   Flare’s FBA achieves safety without relying too much on economic incentives that can interfere with high-value use cases.
+   In Flare, each node has its own local set of validators or quorum set also known as Unique Node List (UNL).
+   This varies from the usual FBA because of another requirement that there should be a minimum level of overlap between UNLs.
+   This makes the network more single node failure resistant as well as Sybil attack resistant.
+   This is because a malicious actor is unlikely to be able to spin up many node instances that will also make it to the UNL list of other existing quorum sets.
+   The native token, Spark, can be used to delegate to the price providers for FTSO (link to FTSO article) without having any locking thereby enabling other usage of Spark at the same time.
+   There is no hard link between the safety of the network and the value of native token Spark which allows greater flexibility for how Spark can be used.
```

Comment:
I don't think section has a clear narrative to follow. It's a bit a mix of everything, without clear sequence to the explanation. Needs a bit more work.